app/views/staff/transactions/_form.html.haml

```diff
@@ -16,3 +16,7 @@
   = f.govuk_text_field :receiving_organisation_reference
 
 = f.govuk_submit t("default.button.submit")
+
+- if action_name == "new"
+  = link_to t("form.link.activity.back"), organisation_activity_path(@activity.organisation, @activity), class: "govuk-button govuk-button--secondary", "data-module": "govuk-button", role: "button"
```

Comment:
I know I've already approved your other PR that also adds this button(!), but I think this should be a [back link](https://design-system.service.gov.uk/components/back-link/) instead.

Reply:
See, I did wonder about that, but the ticket seemed to suggest something that felt more like an action, rather than navigating